src/main/java/io/anserini/ltr/FeatureExtractorCli.java

```diff
@@ -82,18 +82,17 @@
     if (parsedArgs.collection.equals("gov2") || parsedArgs.collection.equals("webxml")) {
       // Open the topics file and read it
       String className = parsedArgs.collection.equals("gov2") ? "Trec" : "Webxml";
-      TopicReader tr = (TopicReader)Class.forName("io.anserini.search.query."+className+"TopicReader")
+      TopicReader<Integer> tr = (TopicReader)Class.forName("io.anserini.search.query."+className+"TopicReader")
```

Comment:
Should be generic type here

Reply:
Why? `className` is either `Trec` or `Webxml` and both use `Integer`.

Reply:
see the gov2 and webxml above?
This piece of code was written years ago and we will definitely support more generic types going forward